test/unit/spec/MfaVerify_spec.js

```diff
@@ -730,12 +730,40 @@ function (Okta,
             var button = test.form.submitButton();
             var buttonClass = button.attr('class');
             expect(buttonClass).toContain('link-button-disabled');
+            expect(button.prop('disabled')).toBe(true);
             return Expect.waitForFormError(test.form, test);
           })
           .then(function (test) {
             var button = test.form.submitButton();
             var buttonClass = button.attr('class');
+            expect($.ajax.calls.count()).toBe(1);
             expect(buttonClass).not.toContain('link-button-disabled');
+            expect(button.prop('disabled')).toBe(false);
+          });
+        });
+        itp('makes only one request if "verify button" was clicked twice', function () {
```

Comment:
`test.setNextResponse([_.extend({delay: 500}, resInvalid), resInvalid]);`
I use the `delay`, so the button is disabled (before the changes it was not disabled, just has class `link-button-disabled`).
And while the request is in progress I click the button the second time.

I can remove this test if needed and left only tests for the `disabled` property.
This test is showing the problem from the issue.